roles/zuul/meta/main.yml

```diff
@@ -1,2 +1,3 @@
 dependencies:
   - pip
+  - logrotate
```

Comment:
When it's directly in the meta file, it cannot be overridden by other deployers, if we have any other deployers. I'm not sure if we want this to be a configurable thing within Hoist, or if it should be static.

Also, I prefer as much of the role configuration to be in the defaults, rather than scattered in other files.